app/web/template/choose_attorneys_summary.gohtml

```diff
@@ -0,0 +1,56 @@
+{{ template "page" . }}
+
+{{ define "main" }}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">{{ trCount .App "attorneysAdded" (len .Lpa.Attorneys) }}</h1>
+
+        <dl class="govuk-summary-list">
+        {{range $i, $a := .Lpa.Attorneys}}
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">
+                    {{ $a.FirstNames }} {{ $a.LastName }}<br>
+                    <span class="govuk-body-s">{{ formatDate $a.DateOfBirth }}</span><br>
+                    <a class="govuk-link govuk-body-s" href="{{ $.AttorneyDetailsPath }}?id={{ $a.ID }}&from=summary">
+                        Change<span class="govuk-visually-hidden"> details for {{ $a.FirstNames }} {{ $a.LastName }}</span>
+                    </a>
+                </dt>
+                <dd class="govuk-summary-list__value">
+                    {{ $a.Address.Line1 }}, {{ $a.Address.Postcode }}<br>
+                    <a class="govuk-link govuk-body-s" href="{{ $.AttorneyAddressPath }}?id={{ $a.ID }}&from=summary">
+                        Change<span class="govuk-visually-hidden"> address for {{ $a.FirstNames }} {{ $a.LastName }}</span>
+                    </a>
+                </dd>
+                <dd class="govuk-summary-list__actions">
+                    <a class="govuk-link govuk-body-s" href="#">
```

Comment:
is there a ticket for this?

Reply:
Will add one now - I'd already bloated this ticket way past where I thought I'd get to so wanted to get some design feedback before adding in another screen.

Reply:
https://opgtransform.atlassian.net/browse/MLPAB-359

Reply:
:+1: definitely smart to cut scope where you can on this